README.md

```diff
@@ -131,6 +131,23 @@ The command to type in Firefox address bar will become `n <package> [destination
 <s>If you use [duckduckgo.com](https://duckduckgo.com) as your primary search engine, type `!njt <package> [destination]` in its search field (note the leading exclamation mark).
 This trick is possible thanks to DuckDuckGo’s awesome [bang feature](https://duckduckgo.com/bang).</s>
 
+### 🟢 Alfred web search
+
+Want to hop directly from [Alfred launcher](https://www.alfredapp.com/)?
+
+1. Open Preferences -> Features -> Web Search
+2. Click "Add Custom Search"
+3. Fill in the blanks like so:
+  * Search URL: `https://njt.now.sh/jump?to={query}`
+  * Title: `Search NJT for '{query}'`
```

Comment:
I am afraid to say something nosy, but `njt` in the lowercase is nicer here and...